packages/react-components/src/components/FileCard.tsx

```diff
@@ -99,31 +128,40 @@ export const _FileCard = (props: _FileCardProps): JSX.Element => {
   };
 
   return (
-    <Stack
-      className={containerClassName}
-      onClick={() => {
-        props.actionHandler && props.actionHandler();
-      }}
-    >
-      <Stack horizontal horizontalAlign="space-between" verticalAlign="center" className={fileInfoWrapperClassName}>
-        <Stack>
-          {/* We are not using <ChatCompositeIcon /> here as we currently do not support customizing these filetype icons. */}
-          <Icon
-            {...getFileTypeIconProps({
-              extension: fileExtension,
-              size: 24,
-              imageFileType: 'svg'
-            })}
-          />
-        </Stack>
-        <Stack className={fileNameContainerClassName}>
-          <Text className={fileNameTextClassName}>{fileName}</Text>
-        </Stack>
-        <Stack verticalAlign="center" className={actionIconClassName}>
-          {actionIcon && actionIcon}
+    <>
+      <Announcer announcementString={announcerString} ariaLive={'polite'} />
```

Comment:
How would this behave if someone uploaded 10 OR 50 files?
I am assuming the announcer would go crazy :D

Reply:
the polite tag should make it wait until other more important announcements are done and announce the current progress. 

Though I wonder if this is indeed something we want. did the A11y team ask for progress percentage announcements?

Reply:
@anjulgarg the announcements are made in sequence of file uploads. Yes, there is a bug filed for this issue https://skype.visualstudio.com/SPOOL/_workitems/edit/2864679